pest/src/error.rs

```diff
@@ -43,9 +42,11 @@ pub struct Error<R> {
     parse_attempts: Option<ParseAttempts<R>>,
 }
 
+#[cfg(feature = "std")]
+impl<R: RuleType> core::error::Error for Error<R> {}
```

Comment:
oh awkward mix of `std` and `core` here, but worth asking. Would you be fine bumping the MSRV up to 1.81.0 for `core::error::Error`? It's only one version up and I believe debian's next release will be using 1.84